test/cpp/src/092-ptw-builder.cc

```diff
@@ -9,7 +9,7 @@
 #include "channel.h"
 
 TEST_CASE("The MSHR factor uses the number of upper levels to determine the PTW's default number of MSHRs") {
-  MEMORY_CONTROLLER dram{champsim::chrono::picoseconds{3200}, champsim::chrono::picoseconds{6400}, std::size_t{18}, std::size_t{18}, std::size_t{18}, std::size_t{38}, champsim::chrono::microseconds{64000}, {}, 64, 64, 1, champsim::data::bytes{8}, 1, 128, 1, 1, 1, 8192};
+  MEMORY_CONTROLLER dram{champsim::chrono::picoseconds{3200}, champsim::chrono::picoseconds{6400}, std::size_t{18}, std::size_t{18}, std::size_t{18}, std::size_t{38}, champsim::chrono::microseconds{64000}, {}, 64, 64, 1, champsim::data::bytes{8}, 1024, 1024, 4, 4, 4, 8192};
```

Comment:
What is the purpose of these edits? They don't seem to have anything to do with the patch.